build.Dockerfile

```diff
@@ -6,7 +6,7 @@ RUN go get github.com/Masterminds/glide \
 && go get github.com/kisielk/errcheck
 
 # Which docker version to test on
-ENV DOCKER_VERSION 1.10.1
+ENV DOCKER_VERSION 1.9.1
```

Comment:
Maybe we should use multiple versions in travis tests: https://docs.travis-ci.com/user/environment-variables/#Global-Variables